app/assets/stylesheets/components/common/_contact_panel.scss

```diff
@@ -214,4 +236,33 @@
       }
     }
   }
+
+  .contact-panel__chat-closed {
+    .contact-panel__dynamic-content {
+      &.is-loading {
+        .contact-panel__chat-description,
+        .contact-panel__button {
+          visibility: visible;
+        }
+        .contact-panel__preloader {
+          display: none;
+        }
+      }
+    }
+  }
+}
+
+// N.B. This is to over-ride styles from Yeast
+// Should be added to that for a final deployment
+.l-contact-panel {
+  width: 100%;
+  margin: 0;
+
+  @include respond-to($mq-l) {
+    @include column(6);
+  }
 }
```

Comment:
Now that the styles for l-contact-panel are in Yeast can we delete them from frontend (along with the comments).